examples/gw_eccentricity_demo.ipynb

```diff
@@ -28,7 +28,7 @@
     "from gw_eccentricity import measure_eccentricity\n",
     "from gw_eccentricity import get_available_methods\n",
     "from gw_eccentricity.plot_settings import use_fancy_plotsettings, labelsDict\n",
-    "use_fancy_plotsettings()    # for better looking plots"
+    "use_fancy_plotsettings(usetex=False)    # for better looking plots"
```

Comment:
Remove the comment, because it sounds like usetex=False leads to better-looking plots.